src/app/home/home.component.ts

```diff
@@ -116,19 +123,39 @@ export class HomeComponent implements OnInit {
     currentZoom: number;
     previousZoom: number;
 
+    public mapFilterForm: FormGroup;
+
+    private displayedSites: Subject<Site[]> = new Subject<Site[]>();
+    private setDisplayedSites;
+
     // TODO:1) populate table of events using pagination. consider the difference between the map and the table.
     //      2) setup a better way to store the state of the data - NgRx.This ought to replace storing it in an object local to this component,
     //       but this local store ok for the short term. The data table should be independent of that data store solution.
     constructor(
         private eventsService: EventsService,
+        private formBuilder: FormBuilder,
         private statesService: StatesService,
         private networkNamesService: NetworkNamesService,
         private sensorTypesService: SensorTypesService,
         public currentUserService: CurrentUserService,
         private sitesService: SitesService,
         private displayValuePipe: DisplayValuePipe,
         public snackBar: MatSnackBar
-    ) {}
+    ) {
+        this.mapFilterForm = formBuilder.group({
+            eventsControl: new FormControl(),
+            networkControl: new FormControl(),
+            sensorControl: new FormControl(),
+            stateControl: new FormControl(),
+            surveyedControl: new FormControl(),
+            HWMOnlyControl: new FormControl(),
+            sensorOnlyControl: new FormControl(),
+            surveyedOnlyControl: new FormControl(),
+            bracketSiteOnlyControl: new FormControl(),
+            RDGOnlyControl: new FormControl(),
+            OPDefinedControl: new FormControl(),
+        });
+    }
```

Comment:
@esmyers  I think these can be defined more succinctly, with the `FormControl()` part implied. In WHISPers I did it like this:
```
this.eventSubmissionForm = this.formBuilder.group({
      event_reference: '',
      event_type: [null, Validators.required],
      complete: false,
      public: [true, Validators.required],
      // NWHC only
      staff: null,
      event_status: '1',
      quality_check: { value: null, disabled: true },
      legal_status: 1,
      legal_number: '',
...
```
But maybe you are doing something different that requires this syntax?